src/app/(proper_react)/redesign/(authenticated)/user/dashboard/page.tsx

```diff
@@ -33,7 +40,45 @@ export default async function DashboardPage() {
     return redirect("/redesign/user/welcome/");
   }
 
+  // This contains the latest scan results in our database.
   const latestScan = await getLatestOnerepScanResults(profileId);
+
+  // Attempt to fetch the current scan results from the provider.
+  // If anything is newer, add it to the database.
+  // Continue if there are any errors.
+  try {
+    const currentScan = await getAllScanResults(profileId);
+    const newScanResults: ScanResult[] = [];
+    currentScan.forEach((current) => {
+      latestScan.results.forEach((latest) => {
+        console.debug(new Date(current.updated_at), latest.updated_at);
```

Comment:
Forgot to remove? (Or alternatively, to add more detail to.)

```suggestion
```

Reply:
Yeah forgot to remove, thanks!